cmd/mailbox-controller/controller.go

```diff
@@ -204,103 +194,113 @@ func (ctl *mbCtl) sync1(ctx context.Context, ref any) {
-		logger.Error(err, "Failed to lookup SyncTargets by mailbox workspace name", "mbwsName", mbwsName)
+		logger.Error(err, "Failed to lookup SyncTargets by mailbox space name", "mbsName", mbsName)
 		return false
 	}
 	var syncTarget *edgev2alpha1.SyncTarget
 	if len(byIndex) == 0 {
 	} else {
 		syncTarget = byIndex[0].(*edgev2alpha1.SyncTarget)
 		if len(byIndex) > 1 {
-			logger.Error(nil, "Impossible: more than one SyncTarget fetched from index; using the first", "mbwsName", mbwsName, "fetched", byIndex)
+			logger.Error(nil, "Impossible: more than one SyncTarget fetched from index; using the first", "mbsName", mbsName, "fetched", byIndex)
 		}
 	}
-	workspace, err := ctl.workspaceScopedLister.Get(mbwsName)
+	space, err := ctl.spaceLister.Get(mbsName)
 	if err != nil && !k8sapierrors.IsNotFound(err) {
-		logger.Error(err, "Unable to Get referenced Workspace", "mbwsName", mbwsName)
+		logger.Error(err, "Unable to Get referenced space", "mbsName", mbsName)
 		return true
 	}
 	if syncTarget == nil || syncTarget.DeletionTimestamp != nil {
-		if workspace == nil || workspace.DeletionTimestamp != nil {
-			logger.V(3).Info("Both SyncTarget and Workspace are absent or deleting, nothing to do", "mbwsName", mbwsName)
+		if space == nil || space.DeletionTimestamp != nil {
+			logger.V(3).Info("Both SyncTarget and Mailbox space are absent or deleting, nothing to do", "mbsName", mbsName)
 			return false
 		}
-		err := ctl.workspaceScopedClient.Delete(ctx, mbwsName, metav1.DeleteOptions{Preconditions: &metav1.Preconditions{UID: &workspace.UID}})
+		err := ctl.spaceManagementClient.SpaceV1alpha1().Spaces(ctl.spaceProviderNs).Delete(ctx, mbsName, metav1.DeleteOptions{})
 		if err == nil || k8sapierrors.IsNotFound(err) {
-			logger.V(2).Info("Deleted unwanted workspace", "mbwsName", mbwsName)
+			logger.V(2).Info("Deleted unwanted space", "mbsName", mbsName)
 			return false
 		}
-		logger.Error(err, "Failed to delete unwanted workspace", "mbwsName", mbwsName)
+		logger.Error(err, "Failed to delete unwanted space", "mbsName", mbsName)
 		return true
 	}
 	// Now we have established that the SyncTarget exists and is not being deleted
-	if workspace == nil {
+	if space == nil {
 		_, stOriginalName, _, err := kbuser.AnalyzeObjectID(syncTarget)
 		if err != nil {
-			logger.Error(err, "object does not appear to be a provider's copy of a consumer's object", "syncTarget", syncTarget.Name)
+			logger.Error(err, "Object does not appear to be a provider's copy of a consumer's object", "syncTarget", syncTarget.Name)
 			return false
 		}
-		ws := &tenancyv1alpha1.Workspace{
+		// create the mailbox space
+		mbspace := &spacev1alpha1.Space{
 			ObjectMeta: metav1.ObjectMeta{
 				Annotations: map[string]string{SyncTargetNameAnnotationKey: stOriginalName},
-				Name:        mbwsName,
+				Name:        mbsName,
+			},
+			Spec: spacev1alpha1.SpaceSpec{
+				SpaceProviderDescName: ctl.spaceProvider,
+				Type:                  spacev1alpha1.SpaceTypeManaged,
```

Comment:
I think this will be the most statically common value for Type; could it be made the default?